lib/active_model/serializer/adapter.rb

```diff
@@ -21,9 +21,11 @@ def serializable_hash(options = nil)
       end
 
       def as_json(options = nil)
-        hash = serializable_hash(options)
-        include_meta(hash) unless self.class == FlattenJson
-        hash
+        serializable_hash(options).tap do |hash|
```

Comment:
@joaomdmoura  OT: Might it make more sense for the FlattenJson adapter to implement a no-op `include_meta` so the superclass doesn't need to know about the subclass?